include/aspect/material_model/rheology/strain_dependent.h

```diff
@@ -208,6 +223,14 @@ namespace aspect
            */
           std::vector<double> viscous_strain_weakening_factors;
 
+          /**
+           * The four temperatures that parameterize the temperature-activated strain softening.
```

Comment:
Add a sentence why you need `std::vector<double>` instead of `double`.  I guess because you have different temperature for different compositions and phases?